src/defaultsettings.cpp

```diff
@@ -69,6 +69,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("enable_build_where_you_stand", "false" );
 	settings->setDefault("3d_mode", "none");
 	settings->setDefault("3d_paralax_strength", "0.025");
+	settings->setDefault("3d_convergence_distance", "1.95");
```

Comment:
not in minetest.conf.example, in some setting file, I think it's builtin/defaultsettings.txt